test/functional/apps/discover/group3/_request_counts.ts

```diff
@@ -137,37 +135,35 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         });
       });
 
-      if (!skipSavedSearchTests) {
-        it('should send 2 requests for saved search changes', async () => {
-          await setQuery(query1);
-          await queryBar.clickQuerySubmitButton();
-          await PageObjects.timePicker.setAbsoluteRange(
-            'Sep 21, 2015 @ 06:31:44.000',
-            'Sep 23, 2015 @ 00:00:00.000'
-          );
-          await waitForLoadingToFinish();
-          // creating the saved search
-          await expectSearches(type, 2, async () => {
-            await PageObjects.discover.saveSearch(savedSearch);
-          });
-          // resetting the saved search
-          await setQuery(query2);
-          await queryBar.clickQuerySubmitButton();
+      it('should send 2 requests for saved search changes', async () => {
+        await setQuery(query1);
+        await queryBar.clickQuerySubmitButton();
+        await PageObjects.timePicker.setAbsoluteRange(
+          'Sep 21, 2015 @ 06:31:44.000',
+          'Sep 23, 2015 @ 00:00:00.000'
+        );
+        await waitForLoadingToFinish();
+        // creating the saved search
+        await expectSearches(type, 2, async () => {
+          await PageObjects.discover.saveSearch(savedSearch);
+        });
+        // resetting the saved search
+        await setQuery(query2);
+        await queryBar.clickQuerySubmitButton();
+        await waitForLoadingToFinish();
+        await expectSearches(type, 2, async () => {
+          await PageObjects.discover.clickResetSavedSearchButton();
+        });
+        // clearing the saved search
+        await expectSearches('ese', 2, async () => {
```

Comment:
This `expectSearches` call is hardcoded to `ese` since Discover gets reset to data view mode after pressing `New` from a SQL saved search.